deploy/manifests/dev/us-east-2/tenant/storetheindex/kustomization.yaml

```diff
@@ -3,3 +3,12 @@ kind: Kustomization
 
 resources:
   - ../../../../base/storetheindex
+
+patchesStrategicMerge:
+  - patch-indexer.yaml
+
+configMapGenerator:
+  - name: indexer-env-vars
+    behavior: merge
+    literals:
+      - STORETHEINDEX_PRIV_KEY_PATH=/identity/identity.key
```

Comment:
What creates that key file in the first place?

Reply:
I have generated it manually, encrypted it and checked it into code as a secret. 
That secret is then mounted as a volume on `/identity` path.